extensions/interactions/GraphInput/validator.js

```diff
@@ -53,6 +56,46 @@ oppia.filter('oppiaInteractiveGraphInputValidator', [
     warningsList = warningsList.concat(
       baseInteractionValidationService.getAllOutcomeWarnings(
         answerGroups, defaultOutcome, stateName));
+    for (var i = 0; i < answerGroups.length; i++) {
+      var ruleSpecs = answerGroups[i].rule_specs;
+      for (var j = 0; j < ruleSpecs.length; j++) {
+        var ruleSpec = ruleSpecs[j];
+        try {
+          if (ruleSpec.rule_type === 'IsIsomorphicTo' &&
+              ruleSpec.inputs.g.vertices.length > ISOMORPHISM_VERTICES_LIMIT) {
+            exceedsIsomorphismVerticesLimit = true;
+          }
+
+          if (ruleSpec.inputs.g.vertices.length > VERTICES_LIMIT) {
+            exceedsVerticesLimit = true;
+          }
+        } catch (e) {
+          warningsList.push({
+            type: WARNING_TYPES.CRITICAL,
+            message: (
+              'The rule ' + String(j + 1) +
+              ' in group ' + String(i + 1) + ' is invalid.')
+          });
+        }
+      }
+    }
+
+    if (exceedsIsomorphismVerticesLimit) {
+      warningsList.push({
```

Comment:
I think this and the following warning need to be rule-specific ("The graph used in the rule ... in group ... is ..."), otherwise it's not clear to the creator what exactly to do.